### ISSUE TITLE
Only support for names consistens out of first and last names

### ISSUE THREAD

**Issue by pheyvaer** [openreview/openreview#21]
**Describe the bug**
When adding an author to a new submission you have to provide first and last name while there are people who have names that are not separable in a first and last name. At the moment people are required to either fill in random stuff in one of the fields or split up their name in two parts just so they can fill in the fields.

**To Reproduce**
1. Go to any venue page.
2. Click on 'Add submission'
3. Scroll down to 'Authors'
4. Try to fill in the name of the author.

**Expected behavior**
Have the option to add your name. Optionally, split it into first and last name.

**Screenshots**
In the case below, my colleague his name is "Sitt Min Oo". There is no first name or last name in his name. We are struggling with filling his name in correctly.

![image](https://user-images.githubusercontent.com/9413528/157872178-a9df4b49-43d3-41b9-ba26-b9c19f83b495.png)

**Device information (please complete the following information):**
 - OS: macOS v10.15.7
 - Browser: FireFox
 - Version: 99.0b2 (64-bit)

**Additional context**
https://www.w3.org/International/questions/qa-personal-names
https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/

**Comment by carlosmondra** [comment]
Hi @pheyvaer , thanks for reporting this issue. We are currently working on a solution for this. Unfortunately, it's not something that can be fixed right away as it not only involved code, but also date. For now, the only solution is the one you mentioned: you need to split the name. We know this is not ideal and once we do the change, we will modify the data to fix the issue.

**Comment by JeffFessler** [comment]
On a related note, I am unable to add my middle initial to the profile.  The type-in box is there but it cannot be edited.

**Comment by carlosmondra** [comment]
@JeffFessler, this is a different issue. Currently, names cannot be modified since they are associated to IDs assigned to you automatically. You can add a new name with the middle initial and set it as preferred. This process will create a new ID for you. If you want to further discuss this, please, read the [guidelines](https://github.com/openreview/openreview/blob/main/CONTRIBUTING.md) and create a new issue if necessary. Thanks!

**Comment by JeffFessler** [comment]
Thanks for the helpful reply!  It took me a while to realized that I should add the new name under the profile settings, rather than creating a new profile, but now it works.  Sorry for putting the issue in the wrong place.

**Comment by carlosmondra** [comment]
@pheyvaer I have an update regarding this. It's now possible to only add first name. The last name is not required anymore.
We are discussing whether we can remove the terms first, middle, and last names from the UI. Let me know if you have more feedback and we will continue working on this.

**Comment by pheyvaer** [comment]
Hi @carlosmondra Great to hear the progress! Depending on what you use the data for in the background it might be enough to only have the full name, no? If you have search function to return an author based on their name I guess that having the full name or a name split up in first and last name doesn't change much to the results of such a search function.

**Comment by carlosmondra** [comment]
@pheyvaer that's correct, we are moving towards having a single field (fullname) instead of first, middle, last fields in the profile. However, this will require some data migration and changes in the UI.

**Comment by Jack-bo1220** [comment]
> @JeffFessler, this is a different issue. Currently, names cannot be modified since they are associated to IDs assigned to you automatically. You can add a new name with the middle initial and set it as preferred. This process will create a new ID for you. If you want to further discuss this, please read the [guidelines](https://github.com/openreview/openreview/blob/main/CONTRIBUTING.md) and create a new issue if necessary. Thanks!

Hi, I have a similar question.
what I want to ask is whether one email can register multiple accounts? The reason is that there is a clerical error in the name of my first account, but it seems that the name cannot be edited in the profile. Therefore, I want to register a second account with the same email address.  Or how will I correct the name?
Thank you! @carlosmondra

**Comment by carlosmondra** [comment]
@Jack-bo1220 I see that you already created an issue with this question. Please refer to the [guidelines](https://github.com/openreview/openreview/blob/main/CONTRIBUTING.md) and let's keep the discussion in the issue that you created. Thank you!

**Comment by carlosmondra** [comment]
@pheyvaer, we have stopped using the fields first, middle, and last name for profiles in OpenReview. 

After discussing all the possible alternatives we decided to ask for the full name of anyone that wants to register in OpenReview. All data has been migrated too. 

Thanks for your feedback. I will close the issue now.
